docs/modules/ROOT/pages/documentation/microprofile/openapi.adoc

```diff
@@ -67,7 +67,7 @@ Operations that want to specify an alternative list of servers must define an `o
 | `mp.openapi.schema.` a| Prefix of the configuration property to specify an alternative list of servers to service an operation.
 Prefix of the configuration property to specify a schema for a specific class, in JSON format. The remainder of the property key must be the fully-qualified class name. The value must be a valid OpenAPI schema object, specified in the JSON format. The use of this property is functionally equivalent to the use of the `@Schema` annotation on a Java class, but may be used in cases where the application developer does not have access to the source code of a class.
 
-When a `name` key is provided with a string value, the schema will be added to the `schemas` collection in the `components` object of the resulting OpenAPI document using `name`'s value as the key.
+When a `name` key is provided with a string value, the schema will be added to the `schemas` collection in the `components` object of the resulting OpenAPI document using ``name``'s value as the key.
```

Comment:
A single ` only works at the beginning or end of a word. Character ` in the middle of the word needs to be doubled. In this case, the second one is in the middle ( the ' character is treated as a part of the word), so the ` character needs to be doubled, and not only the second one but also the first one.